components/cd-social-links/cd-social-links.html.twig

```diff
@@ -0,0 +1,43 @@
+{{ attach_library('common_design/cd-social-links') }}
+
+<footer class="cd-social-links">
+  {% set shareText = label|render|striptags|default(title|render|striptags) %}
+  {% set shareMessage = 'Global Humanitarian Overview 2021: '|t ~ shareText %}
+  {% set shareUrl = 'https://gho.unocha.org' ~ url %}
+  <div class="cd-social-links__wrapper">
+    <ul>
+      <li>
+        <a class="cd-social-links__link cd-social-links__link--twitter" href="https://twitter.com/intent/tweet?text={{ shareMessage }}%0A%0A{{ shareUrl|url_encode }}">
+          <span class="visually-hidden">{{ 'Share on Twitter'|t }}</span>
+          {{ icon_twitter|raw }}
```

Comment:
That's exactly why. We originally had them in two contexts. [See this page](https://gho.unocha.org/global-trends/displacement-increasing-while-durable-solutions-remain-elusive):

- Top of the page has black icons on white bg.
- Search the page for "Stranded migrants" to see white icons on black bg

Reply:
Note that come August, doing only black/white is also achievable with a sprite, since we can use CSS filter to invert the color and get black display from a white icon.

https://caniuse.com/css-filters

Reply:
We could achieve any color at all by making the graphics red by default (since CSS HSL 1deg is red) and then using a combo of CSS filters to change hue, saturation, lightness.

The default CSS could be adapted to expect red graphics and process accordingly, but also leave the table open to re-color at will, or even provide a few OCHA-approved variants.